### PR TITLE
subtree-push nightly-2024-09-10

### PR DESCRIPTION
**Pull request by ytmimi**
Bumping the toolchain version as part of a git subtree push

current toolchain (nightly-2024-08-17):
  - 1.82.0-nightly (506052d49 2024-08-16)

latest toolchain (nightly-2024-09-10):
  - 1.83.0-nightly (c2f74c3f9 2024-09-09)

r? @calebcartwright

### CODE REVIEW

**Comment by ytmimi** [comment]
Running the [Diff Check](https://github.com/rust-lang/rustfmt/actions/runs/10786442883)

File: src/lib.rs [hunk 1]
```diff
@@ -4,8 +4,8 @@
 #![recursion_limit = "256"]
 #![allow(clippy::match_like_matches_macro)]
 
-#[macro_use]
-extern crate tracing;
+// #[macro_use]
+// extern crate tracing;
```
Comment (calebcartwright) [inline src/lib.rs:8]: why jsut commented out?

**Review by calebcartwright — APPROVED** [review]
one small nit inline. feel free to merge as-is or address and then merge

File: src/lib.rs (hunk 1 above)
Comment (ytmimi) [reply src/lib.rs:8]: @calebcartwright no idea. It was added in https://github.com/rust-lang/rust/pull/129774. I can remove this.

**Pull request merged**